include/deal.II/dofs/dof_accessor.h

```diff
@@ -1751,6 +1751,24 @@ class DoFCellAccessor : public DoFAccessor<dimension_,
   template <class OutputVector, typename number>
   void
   set_dof_values_by_interpolation(
+    const Vector<number> &local_values,
+    OutputVector &        values,
+    const unsigned int    fe_index =
+      DoFHandler<dimension_, space_dimension_>::invalid_fe_index,
+    const bool perform_check = false) const;
```

Comment:
Is the `perform_check` flag documented? I can't see that by scrolling up a few lines.